src/Makefile.am

```diff
@@ -45,15 +43,23 @@ if ENABLE_WALLET
 LIBBITCOIN_WALLET=libbitcoin_wallet.a
 endif
 
+LIBBITCOIN_CRYPTO= $(LIBBITCOIN_CRYPTO_BASE)
+if ENABLE_SSE41
+LIBBITCOIN_CRYPTO_SSE41 = crypto/libbitcoin_crypto_sse41.a
```

Comment:
Given this and `LIBBITCOIN_CRYPTO_AVX2` aren't referenced elsewhere, they could be inlined.

Reply:
Sure, but I'd prefer to stick with our convention of naming everything for easier re-use down the road.